cluster-scope/components/project-admin-rolebindings/ray-bu/rbac.yaml

```diff
@@ -9,4 +9,4 @@ roleRef:
 subjects:
     - apiGroup: rbac.authorization.k8s.io
       kind: Group
-      name: ray-bu
+      name: ray-bu
```

Comment:
Same linting issue:

```suggestion
      name: ray-bu

```

Reply:
It seems the change to this file can be discarded as a whole.